pkg/network/encoding/encoding_test.go

```diff
@@ -521,7 +521,7 @@ func TestHTTPSerializationWithLocalhostTraffic(t *testing.T) {
 		},
 		AgentConfiguration: &model.AgentConfiguration{
 			NpmEnabled: false,
-			TsmEnabled: false,
+			UsmEnabled: false,
```

Comment:
I had to copy this from #15143 because of an incompatible change made in DataDog/agent-payload#223.